buildSrc/src/main/kotlin/RemotePublishing.kt

```diff
@@ -110,6 +110,14 @@ fun Project.enablePublishing(defaultJars: Boolean = true) {
             withSourcesJar()
         }
 
+        afterEvaluate {
```

Comment:
just highlighting that `afterEvaluate` is considered an anti-pattern and `findByName` is eager (see https://docs.gradle.org/current/userguide/task_configuration_avoidance.html)

but I don't know of a better way to accomplish this

Reply:
I think we have to use `afterEvaluate` because of the protobuf-gradle-plugin, unfortunately.